spec/spec_helper.rb

```diff
@@ -14,6 +14,7 @@ def default_arguments(args = {})
     end
   end
 end
+Dir[Rails.root.join('app/graphql/instrumenters/*.rb')].each { |f| require f }
```

Comment:
Please move lines 10-17 into a new file in spec/support/. If we don't keep such things out of the spec_helper, it will be very crowded.